pkg/client/client.go

```diff
@@ -382,7 +382,7 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 	}
 
 	if sr.ClusterMesh != nil {
-		fmt.Fprintf(w, "ClusterMesh:\t%d/%d clusters ready, %d global-services\n",
+		fmt.Fprintf(w, "ClusterMesh:\t%d/%d remote clusters ready, %d global-services\n",
 			numReadyClusters(sr.ClusterMesh), len(sr.ClusterMesh.Clusters), sr.ClusterMesh.NumGlobalServices)
 
 		for _, cluster := range sr.ClusterMesh.Clusters {
```

Comment:
The word `failures` should be changed to `reconnections` at line 390 (sorry, I cannot comment there).

Reply:
Thanks! Should be fixed now.